blueprint/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Beta: Update the presence of a Microsoft Teams user based upon an inbound interaction
+title: "Beta: Update the presence of a Microsoft Teams user based upon an inbound interaction"
```

Comment:
@DannaShirley @carnellj please ensure YAML is valid. This broke the dev center build.